Simplified.xcodeproj/project.pbxproj

```diff
@@ -5771,7 +5771,7 @@
 				CLANG_WARN_NON_LITERAL_NULL_CONVERSION = YES_ERROR;
 				CODE_SIGN_ENTITLEMENTS = Simplified/SimplyE.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
-				CURRENT_PROJECT_VERSION = 7;
+				CURRENT_PROJECT_VERSION = 0;
```

Comment:
this is now obsolete since we are already at build 3 for OE 2.5.1 on develop, I think. Which makes me think that it's also weird that GitHub reports 2.5.1 as a change, since it's already on develop.

Do you want to verify/update (or remove) the version.build numbers and just merge this PR? there are just minor changes in the Italian localization.

Reply:
Updated the Italian wordings and removed the version.build numbers. However, I will wait until the audiobook feature branch is merged, to avoid conflicts on the other PR.